webapp/channels/src/components/logged_in/logged_in.tsx

```diff
@@ -164,44 +149,22 @@ export default class LoggedIn extends React.PureComponent<Props> {
         GlobalActions.emitBrowserFocus(false);
     }
 
-    // listen for messages from the desktop app
-    // TODO: This needs to be deprecated in favour of a more solid Desktop App API.
-    private onDesktopMessageListener = (desktopMessage: DesktopMessage) => {
+    private updateActiveStatus = (userIsActive: boolean, idleTime: number, manual: boolean) => {
         if (!this.props.currentUser) {
             return;
         }
-        if (desktopMessage.origin !== window.location.origin) {
-            return;
-        }
 
-        switch (desktopMessage.data.type) {
-        case 'register-desktop': {
-            // Currently used by calls
-            const {version} = desktopMessage.data.message;
-            if (!window.desktop) {
-                window.desktop = {};
-            }
-            window.desktop.version = semver.valid(semver.coerce(version));
-            break;
```

Comment:
This doesn't seem handled in the changes of this file. Has it been moved somewhere else?

Reply:
Right here ;)
https://github.com/mattermost/mattermost/blob/d6fc7253397f24992b3109e40581d48074c227b2/webapp/channels/src/utils/desktopAPI.ts#L36